src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -30,73 +30,47 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfo
         {
             private const string AttributeSuffix = "Attribute";
 
-            private readonly Document _document;
+            private readonly SymbolicRenameInfo _info;
+
+            private Document Document => _info.Document!;
             private readonly CodeCleanupOptionsProvider _fallbackOptions;
             private readonly IEnumerable<IRefactorNotifyService> _refactorNotifyServices;
 
             /// <summary>
             /// Whether or not we shortened the trigger span (say because we were renaming an attribute,
             /// and we didn't select the 'Attribute' portion of the name).
             /// </summary>
-            private readonly bool _isRenamingAttributePrefix;
+            private bool IsRenamingAttributePrefix => _info.IsRenamingAttributePrefix;
 
             public bool CanRename { get; }
-            public string? LocalizedErrorMessage { get; }
+            public string? LocalizedErrorMessage => null;
```

Comment:
this was always returning null.  changed to computed property versus field-prop to convey that.